system-parachains/bridge-hubs/bridge-hub-kusama/src/weights/xcm/mod.rs

```diff
@@ -230,8 +230,7 @@ impl<Call> XcmWeightInfo<Call> for BridgeHubKusamaXcmWeight<Call> {
 		XcmGeneric::<Runtime>::clear_topic()
 	}
 	fn alias_origin(_: &Location) -> Weight {
-		// XCM Executor does not currently support alias origin operations
-		Weight::MAX
+		XcmGeneric::<Runtime>::alias_origin()
```

Comment:
@acatangiu shouldn't we also change this for the relays? E.g. here: https://github.com/polkadot-fellows/runtimes/blob/main/relay/polkadot/src/weights/xcm/mod.rs#L264-L267

Reply:
I was thinking that with AHM we are moving away from user actions on the Relay Chain. After AHM, I don't see real use cases for this feature on RCs, so I prefer to keep it disabled.

Reply:
> We could also use bench bot and generate just XCM/pallet-xcm benchmarks to see if we have everything

Sure. Would you be willing to run the commands here directly if you already know them?

Reply:
We need this: https://github.com/acatangiu/runtimes/pull/2